docs/api/1.x/pagination.rst

```diff
@@ -0,0 +1,29 @@
+Paginate
+########
+
+If the ``_limit`` parameter is provided, the number of records returned is limited.
```

Comment:
For example: Plural endpoints (such as collections) support *limitation* of number returned objects